src/rpcq.lisp

```diff
@@ -374,11 +374,9 @@ class ~A(Message):
 CoreMessages.~A = _deprecated_property(~:*~A)
 " (symbol-name msg-name))))))
 
-(defgeneric serialize (obj stream)
-  (:documentation "Serialize OBJ and append its representation to STREAM"))
-
-(defmethod serialize (obj stream)
-  (yason:encode obj stream))
+(defun serialize (obj)
+  "Serialize OBJ and append its representation to STREAM"
```

Comment:
```suggestion
  "Serialize OBJ and write its representation to STREAM."
```

Reply:
Really this should omit the mention of STREAM, which is no longer a parameter.